### ISSUE TITLE
NBT/Tag Dupe

### ISSUE THREAD

**Issue by SteeltonMC** [Prunt/RestrictedCreative#87]
Hello Mr. Plugin Developer. I come here asking for help and a feature request. The DEV of the plugin this is happening to is unwilling to assist me, so I really hope you take this into consideration.

It boils down to requesting when certain items have a special Bukkit Tag from a plugin called "Soulbind"]

When a player picks up an item that is not Soulbind to them, it gets dropped while in Creative.

I am requesting a blacklist for Item Tags, and when an item is held in creative, with said tag, it is destroyed.

Please respond back with any further questions, or suggestions. It would mean a lot!
_________________________________________________________________________________________

How to replicate

`so now to tell how it happens

put a soulbound item into an item frame, pick block it with middle click in limited creative

save that item to a creative hotbar and bring it to a singleplayer world

delete all the enchants and damage values, and put the nbt data thats left on any item of your choice

using mcedit is how i figured it out

copy the new item onto a creative hotbar and go back into multiplayer

go into limited creative and simply hold the item while standing on a hopper

if its a spawn egg you can rig that up with custom entities like falling  command blocks complete with /op @a

and you can fire it from a dispenser`

TLDR: Request to block or blacklist all non-vanilla bukkit/item tags.

**Comment by hankur** [comment]
What version of Spigot and RestrictedCreative are you using?

**Comment by SteeltonMC** [comment]
Latest paper version for 1.15.2

Latest RC update. Would you like me to try and explain the issue/details more in depth? I personally don't understand the mechanics, a couple people on my server found the bug. It's a bit complex.

**Comment by hankur** [comment]
I think I understand the problem. However, this GitHub page is for issues with RC v1.x, not v2.x. I'll take a look and reply to your post on SpigotMC plugin Discussion page.
